packages/codemirror-extension/src/index.ts

```diff
@@ -348,7 +348,7 @@ function activateEditorCommands(
       }
       modeMenu.addItem({
         command: CommandIDs.changeMode,
-        args: { ...spec }
+        args: { ...spec } as any // TODO: Casting to `any` until lumino typings are fixed
```

Comment:
Xref this comment: https://github.com/jupyterlab/lumino/pull/32#issuecomment-567958901

Reply:
PR: https://github.com/jupyterlab/lumino/pull/33